components/script/dom/webidls/Body.webidl

```diff
@@ -7,6 +7,7 @@
 [Exposed=(Window,Worker)]
 interface mixin Body {
   readonly attribute boolean bodyUsed;
+  readonly attribute object? body;
```

Comment:
I forget; have we established what it would take to use `ReadableStream` here?

Reply:
We haven't established it yet.